Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 // this will need to be pipeline-library@master when the pr merges
-@Library('pipeline-library') _
+@Library('pipeline-library@post-release-step-creds-for-github') _
```

Comment:
I'm going to test this branch out when we go to release v5.0.6
https://bitbucket.org/inindca/pipeline-library/branch/post-release-step-creds-for-github#Lsrc/com/genesys/pipelines/WebappUtils.groovyT306